Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -179,35 +179,35 @@ final class BolusEntryViewModel: ObservableObject {
 
     func saveAndDeliver(onSuccess completion: @escaping () -> Void) {
         guard dataManager.pumpManager != nil else {
-            activeAlert = .noPumpManagerConfigured
+            presentAlert(.noPumpManagerConfigured)
             return
         }
 
         guard let maximumBolus = maximumBolus else {
-            activeAlert = .noMaxBolusConfigured
+            presentAlert(.noMaxBolusConfigured)
             return
         }
 
         guard enteredBolus < maximumBolus else {
-            activeAlert = .maxBolusExceeded
+            presentAlert(.maxBolusExceeded)
             return
         }
 
         if let manualGlucoseSample = manualGlucoseSample {
             guard Self.validManualGlucoseEntryRange.contains(manualGlucoseSample.quantity) else {
-                activeAlert = .manualGlucoseEntryOutOfAcceptableRange
+                presentAlert(.manualGlucoseEntryOutOfAcceptableRange)
                 return
             }
 
             isInitiatingSaveOrBolus = true
-            dataManager.loopManager.glucoseStore.addGlucose(manualGlucoseSample) { result in
+            dataManager.loopManager.addGlucose([manualGlucoseSample]) { result in
```

Comment:
(I realized that LoopDataManager has an API that already calls in to the GlucoseStore API, better to just use that one. Tiny change so I'm tossing it in here.)